### ISSUE TITLE
Strip link to specific results file for end users?

### ISSUE THREAD

**Issue by bloodearnest** [opensafely-core/interactive-templates#115]
Based on Brian's latest user video

**Comment by LFISHER7** [comment]
I agree with removing this. We provide this info to output checkers in the output checking template anyway

**Comment by CLStables** [comment]
We should remove this link at some point, but this isn't urgent (so doesn't need to be done in v2)
